Makefile

```diff
@@ -75,6 +75,6 @@ jsonnet:
 	jsonnet -J /ksonnet-lib hack/generate/prometheus-operator-rbac.jsonnet | json2yaml > contrib/kube-prometheus/manifests/prometheus-operator/prometheus-operator.yaml
 
 jsonnet-docker:
-	docker build -f scripts/jenkins/jsonnet/Dockerfile -t po-jsonnet .
+	docker build -f scripts/Dockerfile -t po-jsonnet .
```

Comment:
Might be easier to understand the purpose of this dockerfile if it is inside a folder like `jsonnet`?